Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:13.12-alpine as build
+WORKDIR /usr/src/app
+
+COPY ["package.json", "package-lock.json", "./"]
+RUN npm install --silent
+COPY . .
+RUN npm run build
+
+FROM node:13.12-alpine
+WORKDIR /app
+COPY ["package.json", "package-lock.json", "./"]
+COPY --from=build /usr/src/app/dist /app/dist
+RUN npm install --silent

```

Comment:
You should add a `--production` flag (or `NODE_ENV` environment variable to production) so npm will not install modules listed in `devDependencies`.